lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= The Distributive Property
+
+@description{Students develop an understanding of the Distributive Property by studying equivalent Circles of Evaluation. They then apply their understanding of the Distributive Property in a variety of contexts.}
+
+@lesson-prereqs{identity-property}
+
+@keywords{Circle of Evaluation, }
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Recognize how distribution results in equivalent expressions.
+* Apply the Distributive Property to expressions with numbers and variables.
+* Use distribution to efficiently complete mental computations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Distributive Property and Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* A common approach to teaching the Distributive Property is by using an area model. This approach works well for students who are comfortable visualizing and computing area - but may leave other students confused.
+* We introduce the Distributive Property by inviting students to look at patterns in _structure_, and then later apply what they know using the area model.
```

Comment:
This second bullet describes what we're doing. 
I'd suggest leaving out the first bullet. If it feels critical to you, I'd suggest rewording so that it feels like what we're doing is additive, rather than suggesting that it is a mistake to use the area model.